addons/event/static/src/client_action/event_barcode.scss

```diff
@@ -24,6 +24,34 @@
         max-width: 200px;
         max-height: 100px;
     }
+    .o_barcode_mobile_container {
+        transform: scale(2.5);
+        margin-top: 75px;
+        margin-bottom: 20px;
+
+        img {
+            height: 70px;
+        }
+
+        .o_mobile_barcode{
+            opacity: 0;
+            height: 85px;
+            bottom: -15px;
+        }
+        .o_barcode_laser{
+            height: 2px;
+        }
+    }
+    @media (max-width: 750px) {
+        main {
```

Comment:
If we can be more specific it would be nice (for instance with an added selector)